### PR TITLE
Added a docstring to a private method in point.py.

### PR DESCRIPTION
**Pull request by moorepants**
#### Brief description of what is fixed or changed

Was trying to understand this module and wrote this for future reference.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/physics/vector/point.py [hunk 1]
```diff
@@ -68,7 +68,32 @@ def _check_point(self, other):
...
+        Returns
+        =======
+        list of Points
+            A sequence of points from self to other.
+
+        Notes
+        =====
+
+        It isn't clear if num = 1 or num = 2 actually works because the keys to
+        ``_vel_dict`` and ``_acc_dict`` are reference frames and reference
```
Comment (eric-wieser) [inline sympy/physics/vector/point.py:93]:
```suggestion
        ``_vel_dict`` and ``_acc_dict`` are :class:`ReferenceFrame` objects which
```

**Comment by moorepants** [comment]
This is ready for merging. Should be a pretty simple review.

File: sympy/physics/vector/point.py [hunk 2]
```diff
@@ -68,7 +68,32 @@ def _check_point(self, other):
...
+        =======
+        list of Points
+            A sequence of points from self to other.
+
+        Notes
+        =====
+
+        It isn't clear if num = 1 or num = 2 actually works because the keys to
+        ``_vel_dict`` and ``_acc_dict`` are :class:`ReferenceFrame` objects which
+        frames do not have the ``_pdlist`` attribute.
```
Comment (eric-wieser) [inline sympy/physics/vector/point.py:94]:
```suggestion
        ``_vel_dict`` and ``_acc_dict`` are :class:`ReferenceFrame` objects which
        do not have the ``_pdlist`` attribute.
```
Apologies, previous suggestion was on mobile and I couldn't select both lines.

**Comment by eric-wieser** [comment]
There's a nearly identical function in `vector/frame.py` - is it worth either copying the docstring to there or adding a note there to tell the reader to look at the other docstring?

**Comment by moorepants** [comment]
> There's a nearly identical function in vector/frame.py - is it worth either copying the docstring to there or adding a note there to tell the reader to look at the other docstring?

I think this was copied from there and wasn't modified in a way that it works fully with with this class. I'd like to just merge this as is if possible without having to spend time on the other module. I can do that as some point in the future.

**Review by eric-wieser — APPROVED** [review]

**Pull request merged**